scripts/generate-css/styles.ts

```diff
@@ -154,90 +159,85 @@ export const generateStyles = async (
     const docs = generateDocs({ properties: name, urls: [url], deprecated })
 
     type = computedType({ type, token, transform, prop })
-    standardStyles = [...standardStyles, `${prop}: ${config}`]
-    styleProps = [...styleProps, ...[docs, `${prop}?: ${type}`]]
+    standardStyles.push(`${prop}: ${config}`)
+    styleProps.push(...[docs, `${prop}?: ${type}`])
 
     if (shorthands) {
       const shorthandStyle =
         config === true ? `{ properties: "${prop}" }` : `standardStyles.${prop}`
 
       shorthands.forEach((shorthandProp) => {
-        shorthandStyles = [
-          ...shorthandStyles,
-          `${shorthandProp}: ${shorthandStyle}`,
-        ]
-        styleProps = [...styleProps, ...[docs, `${shorthandProp}?: ${type}`]]
+        shorthandStyles.push(`${shorthandProp}: ${shorthandStyle}`)
+        styleProps.push(docs, `${shorthandProp}?: ${type}`)
       })
     }
   })
 
-  Object.entries<UIOptions>(uiProps).forEach(
-    ([
+  const addStyles = (
+    [
       prop,
       {
         properties,
         transform,
         static: css,
         type,
         hasToken,
-        isAtRule,
         isProcessResult,
         isProcessSkip,
-        isSkip,
         description,
       },
-    ]) => {
-      if (isProcessSkip)
-        processSkipProperties = [...processSkipProperties, prop]
-
-      const relatedStyles = styles.filter(({ prop }) =>
-        typeof properties === "string"
-          ? prop === properties
-          : properties?.includes(prop),
-      )
-      const deprecated = relatedStyles.some(({ deprecated }) => deprecated)
-      const urls = relatedStyles.map(({ url }) => url)
-      const types = relatedStyles.map(({ type }) => type)
-      const token = tokenMap[prop as UIProperties]
-      const shorthands = shorthandProps[prop as UIProperties]
-
-      transform ??= transformMap[prop as UIProperties]
-      type = computedType({ type: type ?? types, hasToken, token, transform })
-
-      const config = getConfig({
-        properties,
-        token,
-        transform,
-        isProcessResult,
-        isProcessSkip,
-        isSkip,
-        css,
-      })(true)
+    ]: [string, UIOptions],
+    targetStyles: string[],
+  ) => {
+    if (isProcessSkip) processSkipProperties.push(prop)
+
+    const relatedStyles = styles.filter(({ prop }) =>
+      typeof properties === "string"
+        ? prop === properties
+        : properties?.includes(prop),
```

Comment:
Shouldn't we use `isString` provided by @yamada-ui/utils?

Reply:
@taroj1205 

You cannot use `@yamada-ui/utils` or `@yamada-ui/react` here.